### ISSUE TITLE
Windows build failing with error: 'M_PI' was not declared in this scope

### ISSUE THREAD

**Issue by cgspeck** [Brewtarget/brewtarget#538]
I noticed the Windows build failing with [the following](https://github.com/Brewtarget/brewtarget/runs/2251199271?check_suite_focus=true#step:8:149):

```
[ 58%] Building CXX object src/CMakeFiles/btobjlib.dir/RadarChart.cpp.obj
D:\a\brewtarget\brewtarget\src\RadarChart.cpp:44:40: error: 'M_PI' was not declared in this scope
    double const RadiansInACircle = 2 * M_PI;
                                        ^~~~
mingw32-make[2]: *** [src\CMakeFiles\btobjlib.dir\build.make:1378: src/CMakeFiles/btobjlib.dir/RadarChart.cpp.obj] Error 1
```

https://github.com/Brewtarget/brewtarget/pull/530 introduced the reference to [`M_PI`](https://github.com/Brewtarget/brewtarget/pull/530/commits/00a840ccf8a6ee760ded3eb72e129ba2e9c08091#diff-0705086554f2e8405975713842728f72d4d4d8421cfc5729434f4f9bae10c51bR42). 

Is there a new dependency, step or build flag that needs to be added?

**Comment by matty0ung** [comment]
Ah, looks like we need an extra define on Windows.  (Until we're able to use C++20, there is no "standard" way to get the value of 𝜋 in C++.)  Most compilers add a non-standard constant M_PI into the cmath header, but it looks like Windows needs another define.  It's a one-line fix (plus comment).  I'll do a patch.

**Comment by cgspeck** [comment]
Hi. Thanks but unfortunately the fix did not work - you can see on the Actions Tab / Windows / [Build](https://github.com/Brewtarget/brewtarget/runs/2251621697?check_suite_focus=true#step:8:149) for #539.

Could we define `M_PI` in the file like [this example](https://stackoverflow.com/a/26065433/806799)?

```
#ifndef M_PI
    #define M_PI 3.14159265358979323846
#endif
```

If you don't want to add that, then try putting `_USE_MATH_DEFINES` at the top of the file with the `cmath` include directly after it.

**Comment by matty0ung** [comment]
Ah, yes, sorry, if I'd thought about it, I'd have realised it makes sense to do the _USE_MATH_DEFINES define before including the cmath header..

**Comment by mattiasmaahl** [comment]
This is fixed in #543. closing.
